care/emr/tests/test_questionnaire_response_api.py

```
@@ -0,0 +1,586 @@
+                },
+                {
+                    "id": "286afbde-9ac1-4717-964e-17453fa0ea01",
+                    "text": "FiO2 (%)",
+                    "type": "decimal",
+                    "link_id": "1.3.7",
+                    "required": False,
+                },
+            ],
+            "styling_metadata": {"containerClasses": "grid-2-col"},
+        },
+    ]
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)